PennMobile/About/Privacy/PermissionView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  PrivacyPermissionView.swift
+//  PennMobile
+//
+//  Created by Dominic Holmes on 12/31/19.
+//  Copyright © 2019 PennLabs. All rights reserved.
+//
+import SwiftUI
+import Combine
+
+@available(iOS 13, *)
+struct PermissionView: View {
+    
+    enum Choice {
+        case affirmative, negative, close, moreInfo
+    }
+    
+    @ObservedObject var delegate: PrivacyPermissionDelegate
+    let title: String
+    let privacyString: String
+    let affirmativeString: String
+    let negativeString: String
+    let moreInfoString: String
+    
+    var body: some View {
+        VStack(alignment: .center, spacing: 17) {
+            
+            Image("pennmobile")
+                .resizable()
+                .frame(minWidth: 45, maxWidth: 70, minHeight: 45, maxHeight: 70)
+                .scaledToFit()
+                .padding(.vertical, 17)
+            
+            Text(title)
+                .font(.largeTitle)
+                .fontWeight(.bold)
+            
+            Text(privacyString)
+                .padding()
+                .multilineTextAlignment(.center)
+                .minimumScaleFactor(0.8)
+                .layoutPriority(1)
+            
+            Button(action: moreAboutLabs) {
+                HStack {
+                    Text(moreInfoString)
+                    Image(systemName: "arrow.right.circle")
+                }
+                .font(.system(size: 17, weight: .semibold))
+            }
+            
+            Spacer()
+
+            FullButton(action: shareCourses, content: Text(affirmativeString), foreground: .white, background: .blue)
+            FullButton(action: doNotShareCourses, content: Text(negativeString), foreground: .primary, background: .clear)
+        }
+    }
+    
+    func moreAboutLabs() {
+        delegate.userDecision = .moreInfo
+    }
+    
+    func shareCourses() {
+        delegate.userDecision = .affirmative
+    }
+    
+    func doNotShareCourses() {
+        delegate.userDecision = .negative
+    }
+    
+    func closeView() {
+        delegate.userDecision = .close
+    }
```

Comment:
When is this triggered?

Reply:
its not -- was going to add an X button then decided against it